SETUP.md

```diff
@@ -63,7 +63,8 @@ cd CircuitVerse
 1. Install Ruby bundler : `gem install bundler`
 2. Install Ruby dependencies: `bundle install`
 3. Install Yarn dependencies: `yarn`
-4. Configure your PostgreSQL database in `config/database.yml` (copy `config/database.example.yml` for the template)
+4. Configure your PostgreSQL database in `config/database.yml` (copy `config/database.example.yml` for the template): 
+     * **command:** `cp config/database.example.yml config/database.yml`
```

Comment:
This command work in windows

Reply:
this works on mac too(tried) , also works on linux with -r flag

Reply:
Oops, I made a typo